fresnel/interact.py

```diff
@@ -185,6 +185,8 @@ class SceneView(QWidget):
         - :doc:`examples/02-Advanced-topics/03-Interactive-scene-view`
     """
 
+    update_signal = QtCore.Signal(camera.Camera)
```

Comment:
Good to see this working. Can we rename the signal to `rendering`?

Reply:
of course!